src/EventListener/LastVisitedPageOnGetListener.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace AppBundle\EventListener;
+
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\Event\GetResponseEvent;
+use Symfony\Component\HttpKernel\KernelEvents;
+use Symfony\Component\Security\Core\Authentication\Token\AnonymousToken;
+use Symfony\Component\Security\Core\Security;
+use Symfony\Component\Security\Http\FirewallMapInterface;
+
+class LastVisitedPageOnGetListener implements EventSubscriberInterface
+{
+    private $ignoredRouteNames = ['app_user_login', 'logout', 'app_membership_activate', 'asset_url', '_profiler', '_wdt'];
```

Comment:
you can remove routes that does not belong to the main firewall, don't you ?

Reply:
Done.